src/SplitTuples.cpp

```diff
@@ -53,12 +53,6 @@ class UsesExternImage : public IRVisitor {
     bool result = false;
 };
 
-inline bool uses_extern_image(const Stmt &s) {
```

Comment:
I think this means you can delete the class UsesExternImage too.

Reply:
Done